### ISSUE TITLE
Dash v5 no longer takes you right to the result.

### ISSUE THREAD

**Issue by ryanpcmcquen** [farcaller/DashDoc#70]
Maybe this is a question for @Kapeli instead, but with Dash v5, we no longer get an immediate result when invoking on a selected word.

v4:
<img width="801" alt="Screen Shot 2020-03-18 at 12 03 00 PM" src="https://user-images.githubusercontent.com/772937/76997481-95b0b300-6910-11ea-9d6b-baec33e026ac.png">

---

v5:
<img width="1028" alt="Screen Shot 2020-03-18 at 12 03 29 PM" src="https://user-images.githubusercontent.com/772937/76997496-9ba69400-6910-11ea-9321-efd0ac6b5a82.png">

**Comment by sebastienbarre** [comment]
Yup, same here, I'm not sure what changed in Dash 5 command line parameters? @Kapeli ?

**Comment by farcaller** [comment]
I know it's a meh solution but I'm using dashv4 here 😞

**Comment by ryanpcmcquen** [comment]
Yeah, I'm sticking with v4 until this gets fixed, which is fine, but you'd think @Kapeli would want to fix this so people would want to upgrade.

**Comment by sebastienbarre** [comment]
Hmmm, I need to find v4 then...

**Comment by ryanpcmcquen** [comment]
https://sanfrancisco.kapeli.com/downloads/v4/Dash.zip

**Comment by ryanpcmcquen** [comment]
For anyone curious, this is still true in version 6:
<img width="912" alt="Screen Shot 2022-02-02 at 10 06 39 AM" src="https://user-images.githubusercontent.com/772937/152212091-32e9c50a-1440-4af1-818d-dfa2d463968f.png">

**Comment by ghost** [comment]
The same for 6.4.0 (1006). We just need to make an extra button press/click to get to the resulting doc
